src/Parser/Priest/Discipline/Normalizers/AtonementSuccessiveDamage.js

```diff
@@ -0,0 +1,45 @@
+import EventsNormalizer from 'Parser/Core/EventsNormalizer';
+
+import isAtonement from '../Modules/Core/isAtonement';
+
+class AtonementSuccessiveDamage extends EventsNormalizer {
+
+  normalize(events) {
+    const fixedEvents = [];
+    const _damageEventIndexes = [];
+
+    let _encounteredTargetIDs = [];
+
+    events.forEach((event, eventIndex) => {
+
+      fixedEvents.push(event);
+
+      if(event.type === "damage" && event.sourceIsFriendly && !event.targetIsFriendly){
+        _damageEventIndexes.push(eventIndex);
+        _encounteredTargetIDs = [];
+        return;
+      }
+
+      if(event.type === "heal" && isAtonement(event)) {
+
+        // We encountered a targetID we already encountered since the last damage
+        // event. We push down the last damage event here
+        if(_encounteredTargetIDs.indexOf(event.targetID) >= 0) {
+          const lastDamageEvent = fixedEvents.splice(_damageEventIndexes[_damageEventIndexes.length -1], 1)[0];
```

Comment:
I recommend setting `__modified` to `true` so that this event is highlighted in the events tab:

`lastDamageEvent.__modified = true;`

Reply:
done